ss/pebbledb/db.go

```diff
@@ -354,6 +384,115 @@ func (db *Database) ApplyChangesetAsync(version int64, changesets []*proto.Named
 		Version:    version,
 		Changesets: changesets,
 	}
+
+	if db.config.HashRange > 0 {
+		go func(ver int64) {
+			if err := db.computeMissingRanges(ver); err != nil {
```

Comment:
It seems unnecessary/expensive to call computeMissingRanges every  time we ApplyChangesetAsync?

Shall we do that every DefaultSSHashRange blocks?